tests/installation/bootloader_hyperv.pm

```diff
@@ -31,8 +31,9 @@ sub run {
     my $hyperv_intermediary = select_console('hyperv-intermediary');
     my $name                = $svirt->name;
 
-    # Workaround before fix in svirt (https://github.com/os-autoinst/os-autoinst/pull/879) is deployed
-    set_var('NUMDISKS', defined get_var('RAIDLEVEL') ? 4 : 1);
+    # Workaround before fix in svirt (https://github.com/os-autoinst/os-autoinst/pull/901) is deployed
+    my $n = get_var('NUMDISKS', 1);
+    set_var('NUMDISKS', defined get_var('RAIDLEVEL') ? 4 : $n);
```

Comment:
Let's maybe inject get_var('NUMDISKS', 1) instead of using variable. And as it's used in multiple places, we could also extract it as well. WDYT?

Reply:
Well, frankly I find the set/get spaghetti already complicated enough. I'd also avoid extracting the workaround. Especially when the fix in os-autoinst was merged already, just waiting for the code being deployed on OSD.

Reply:
Good point. Let's merge it then ;)